src/devtools/client/debugger/src/components/Editor/Breakpoints/Panel/Panel.js

```diff
@@ -121,7 +121,9 @@ function Panel({
               toggleEditingOn={toggleEditingOn}
             />
           )}
-          <BreakpointNavigation {...{ breakpoint, editing, showCondition, setShowCondition }} />
+          <BreakpointNavigation
+            {...{ breakpoint, editing, isHot, showCondition, setShowCondition }}
+          />
```

Comment:
Curious to see us using a spread operator here when we're also enumerating props, rather than just listing them the "normal" way :grin:
```js
<BreakpointNavigation
  breakpoint={breakpoint}
  editing={editing}
  isHot={isHot}
  showCondition={showCondition}
  setShowCondition={setShowCondition}
/>
```

Reply:
It's a somewhat common pattern to avoid that annoying prop name repetition :)  Slightly surprised it sounds like you haven't seen it before?

Reply:
I haven't seen it before. Seems kind of wasteful (unnecessary object allocation and spread operator) but to be fair, not in a way that _actually_ matters so 🤷🏼 😆

Reply:
Also just to play devil's advocate against myself– there are _tons_ of objects and functions allocated for every React element rendered so I realize my comment above is silly 😄

Reply:
I left this here because it was already like this but I've never liked this short-hand, personally. I am sympathetic to cutting the repetition, but I just feel like it always looks awkward and adds more cognitive load than just reading the words twice (although it is less typing and less area for typos!). I know this is impossible but I *really* wish that React could interpret an attribute name without an explicit value to the local scope of that token instead of `true`. So:

```
<BreakpointNavigation
  breakpoint
  editing
  isHot
  showCondition
  setShowCondition />
```

There's probably lots of reasons why that is a bad idea but it *feels* right to me 🤷

Reply:
So this is actually a common proposal for a notional "JSX spec 2.0" - the term is "prop punning".  The Rescript language actually does that by default in its JSX variation.

Dan even mentioned this today as a thing that would theoretically happen someday:

https://twitter.com/dan_abramov/status/1523343682476326912

but followed it with "no immediate changes planned, likely more of a “years from now” kind of thing".

Given all that, yes, `<MyComponent {...{prop1, prop2, prop3}} />` is about as close as we're gonna get for a long time.

Reply:
wow I'm surprised it's even being seriously considered, it seems like a pretty huge change! I guess that's what major versions of things are for though.

Reply:
> I know this is impossible but I *really* wish that React could interpret an attribute name without an explicit value to the local scope of that token instead of `true`.

I suspect one reason it doesn't do this is that it would be a backwards breaking change that would be difficult to grep for. (All existing shorthand props would start evaluating as `undefined` instead of `true`.)

Though I think this could be code modded.

Reply:
oh yeah I forgot about typing. That seems like a good enough reason to me.